cmd/earthly/app/run.go

```diff
@@ -385,7 +385,7 @@ func (app *EarthlyApp) run(ctx context.Context, args []string, lastSignal *syncu
 					time.Now(),
 					logstream.FailureType_FAILURE_TYPE_OTHER,
 					"",
-					grpcErr.Message(),
+					messageWithSourceLink(grpcErr.Message()),
```

Comment:
It might make more sense to change the text in the source instead of having to replace it with regex, sense I'm not sure it would only apply to gRPC error message.